src/main/kotlin/LocalEvaluationClient.kt

```diff
@@ -86,56 +68,59 @@ class LocalEvaluationClient internal constructor(
         }.toMap()
     }
 
-    private fun updateRules(): CompletableFuture<Map<String, FlagConfig>> {
-        return doRules().thenApply { newRules ->
-            rulesLock.write {
-                rules = newRules
+    @ExperimentalCohortApi
+    fun enableCohortSync(
+        apiKey: String,
```

Comment:
Is this a different apiKey? And for the usage, this method needs to be called first?

Reply:
The this is the analytics API key and secret key for the project associated with the deployment key used to initialize the local evaluation client. These keys are required to authenticate and provide context for requests to the behavioral cohort API.

This method must be called prior to start. The reason for the weird api is because I dont want to include experimental options on the primary api.